### PR TITLE
Features/cell design

### PR DESCRIPTION
**Pull request by gilemos**
Hey everyone!
I changed the design of the cells and the labels/icons. See if you liked it!

Screen Shot 2019-08-01 at 9 38 39 AM
Screen Shot 2019-08-01 at 9 38 50 AM

### CODE REVIEW

**Comment by gilemos** [comment]
Things that I still want to do:
 - Erase "Places to visit" and put it in the navigation controller
 - Put the button of the lateral table view in the navigation controller
- Change the font of the section titles in home
- Make the last cell of each section of the home table view take us to the more option view controller
- Put some shading in the checkmark button and maybe change the color of the border
- Change the font and color of the navigation controller buttons

File: Travel Scheduler/View Controllers/FirstScreenViewController.m [hunk 1]
```diff
@@ -60,6 +60,7 @@ @interface FirstScreenViewController ()<UISearchBarDelegate, UIPickerViewDelegat
     UITabBarItem *tabBarItem1 = [tabBarController.tabBar.items objectAtIndex:1];
     [tabBarItem1 setImage:[[UIImage imageNamed:@"schedule_icon"] imageWithRenderingMode: UIImageRenderingModeAlwaysOriginal]];
     return tabBarController;
+
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/FirstScreenViewController.m:63]: Why did you add a space here?

**Review by aliu917 — APPROVED** [review]
Looks good!
